bindings/wp8/MegaSDK.cpp

```diff
@@ -1110,11 +1108,14 @@ void MegaSDK::setLogLevel(MLogLevel logLevel)
     MegaApi::setLogLevel((int)logLevel);
 }
 
-void MegaSDK::setLoggerObject(MLoggerInterface^ megaLogger)
+void MegaSDK::addLoggerObject(MLoggerInterface^ megaLogger)
+{
+    MegaApi::addLoggerObject(new DelegateMLogger(megaLogger));
```

Comment:
You need to store the reference passed to `addLoggerObject()`, so you can pass exactly the same instance to `removeLoggerObject()`